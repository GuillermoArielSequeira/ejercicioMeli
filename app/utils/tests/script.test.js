const React = require('react');
const { shallow } = require('enzyme');
const Script = require('../script');

describe('Script Component', () => {
  it('render Script correctly', () => {
    const wrapper = shallow(<Script />);
    expect(wrapper).toMatchSnapshot();
  })
});
