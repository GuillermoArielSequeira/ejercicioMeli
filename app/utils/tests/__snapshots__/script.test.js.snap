// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Script Component render Script correctly 1`] = `ShallowWrapper {}`;
